addons/mail/static/src/activity/activity_menu.scss

```diff
@@ -11,3 +13,12 @@
         background-color: map-get($theme-colors, "light");
     }
 }
+
+.o-mail-no-activity {
+    cursor: initial;
```

Comment:
Don't see any reason for this `cursor: initial` style rule? If unused, then drop.